### ISSUE TITLE
Better handling of temp file

### ISSUE THREAD

**Issue by miguelsousa** [adobe-type-tools/psautohint#65]
When hinting an OTF font, at the end of the process there's a temp file called `<filename>.temp.ac` that gets created in the same directory and then deleted. I'm sure there's a better way of doing whatever needs to be done. The code is in [`otfFont.CFFFontData.saveChanges()`](https://github.com/adobe-type-tools/psautohint/blob/master/python/psautohint/otfFont.py#L1075).

**Comment by khaledhosny** [comment]
Right, I came across this code the other day and suspected the same, but was not sure whether it is worse to raise the required FontTools version for that. But I see this was fixed in 3.0 (older than my recollection) and we require 3.1.2 already.

**Comment by khaledhosny** [comment]
We open the TTFont ourselves, so that shouldn’t be an issue as long as the default does not change.

**Comment by anthrotype** [comment]
It won't change.
